osu.Game.Rulesets.Osu/Difficulty/Evaluators/AimEvaluator.cs

```diff
@@ -34,19 +36,27 @@ public static double EvaluateDifficultyOf(DifficultyHitObject current, bool with
             var osuLastLastObj = (OsuDifficultyHitObject)current.Previous(1);
 
             // Calculate the velocity to the current hitobject, which starts with a base distance / time assuming the last object is a hitcircle.
-            double currVelocity = osuCurrObj.LazyJumpDistance / osuCurrObj.StrainTime;
+            double currVelocity = osuCurrObj.JumpDistance / osuCurrObj.StrainTime;
+
+            // This multiplier exists to prevent slideraim having sliderjumps bonus
+            double sliderJumpsAdjustingMultiplier = 1.0;
```

Comment:
rename to `sliderJumpNerfFactor` or similar. 

also why not have slider body bonus and slider jump bonus coexist? are there any particular cases you can point to where this doesn't make sense?

Reply:
because lost umbrella +100